src/components/Section/Foreign/Activities/IndirectActivity/IndirectInterest.jsx

```diff
@@ -284,9 +284,6 @@ export default class IndirectInterest extends ValidationElement {
             name="Acquired"
             className="acquired"
             {...this.props.Acquired}
-            label={i18n.t(
-              `foreign.activities.indirect.interest.label.acquired`
```

Comment:
Are these still in use, if not could they be removed from the locale definition file as well?

Reply:
These are flagged by the test as being referenced in the code, but missing from the locale file. So in these cases, these strings are unneeded since DateControl doesn't accept a label, so I removed it from the jsx files, and since it's already missing from the locale file, there's no need to remove anything there.